presto-client/src/main/java/com/facebook/presto/client/StatementClient.java

```diff
@@ -132,7 +131,6 @@ private static Request buildQueryRequest(ClientSession session, String query)
         }
         builder.setHeader(PrestoHeaders.PRESTO_TIME_ZONE, session.getTimeZoneId());
         builder.setHeader(PrestoHeaders.PRESTO_LANGUAGE, session.getLocale().toLanguageTag());
-        builder.setHeader(USER_AGENT, USER_AGENT_VALUE);
```

Comment:
I moved this code inside `prepareRequest` https://github.com/prestodb/presto/pull/4929/files#diff-9c9280d592f8582c73302c8114b44a29R222 so as not to forget to set USER_AGENT.